core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -299,6 +306,13 @@ public boolean accepts(@Nullable String acceptHeader) {
 		return produces.accepts(acceptHeader);
 	}
 
+	public boolean isParameterAllowed(@Nonnull String parameterName) {
```

Comment:
Ik weet niet of we misschien ook een hard-coded blacklist moeten toepassen van parameters? Of bij de `configure()` moeten checken of de parameters in de whitelist in de "reserved names" staan?

We hebben nog niet zo'n lijst maar in het ticket heb ik een paar namen van session-keys gezet die we nu in the ApiListenerServlet aanmaken, en die mogelijk via een parameter overschreven zouden kunnen worden. Die lijst zou als basis voor een blacklist / reserved names list kunnen dienen.